src/AutoRest.CSharp/LowLevel/Generation/LowLevelClientWriter.cs

```diff
@@ -75,11 +70,14 @@ private void WriteClientMethod(CodeWriter writer, RestClientMethod clientMethod,
             {
                 writer.WriteParameter(parameter);
             }
-            writer.Line($"{typeof(CancellationToken)} cancellationToken = default)");
+            writer.RemoveTrailingComma();
+            writer.Line($")");
 
             using (writer.Scope())
             {
-                writer.Append($"{typeof(Azure.Core.Request)} req = {RequestWriterHelpers.CreateRequestMethodName(clientMethod.Name)}(");
+                writer.Line($"requestOptions ??= new {typeof(Azure.RequestOptions)}();");
```

Comment:
We should add an `RequestOptions.Default` or something for this case, to save ourselves the allocation on each call.  I don't think that blocks landing this however.

Reply:
Fixed as https://github.com/Azure/azure-sdk-for-net/issues/20913 so we don't forget. I agree this should not block landing.